src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/AldersvilkårVurderingTest.kt

```diff
@@ -55,7 +47,16 @@ class AldersvilkårVurderingTest {
     }
 
     @Test
-    fun `Gyldige perioder med målgruppe DAGPENGER skal gi svar NEI`() {
+    fun `Gyldige perioder med målgruppe OVERGANGSSTØNAD skal kaste feil`() {
```

Comment:
Gir navnet på denne testen mening? Hvis det er gyldig så burde det ikke kaste feil?
`Skal kaste feil ved vurdering av aldersvilkår for målgruppe OVERGANGSSTØNAD` evt?

Reply:
Gjelder nedover også, de er ikke gyldige perioder om de inneholder data de ikke skal ha